cccatalog-api/cccatalog/api/models.py

```diff
@@ -78,7 +78,7 @@ class Image(OpenLedgerModel):
     # Denormalized tags as an array, for easier syncing with Elasticsearch
     tags_list = ArrayField(models.CharField(max_length=255), blank=True, null=True)
 
-    tags = JSONField()
+    tags = JSONField(blank=True, null=True)
```

Comment:
Taggit looks pretty nice, but doesn't apply to this particular use case, since we don't query the database for tags on the Django layer. They are copied to Elasticsearch by the Ingestion Server.